compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -146,6 +146,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val Xlink = BooleanSetting("-Xlink", "Recompile library code with the application.")
   val YoptPhases = PhasesSetting("-Yopt-phases", "Restrict the optimisation phases to execute under -optimise.")
   val YoptFuel = IntSetting("-Yopt-fuel", "Maximum number of optimisations performed under -optimise.", -1)
+  val YnoDecodeStacktraces = BooleanSetting("-Yno-decode-stacktraces", "Show operations that triggered StackOverflows instead of printing the raw stacktraces.")
```

Comment:
Comment is misleading. How I understand it: "if you pass the flag, you won't see original stack traces". I guess it is the opposite

Reply:
Oh right, fixing it